lib/middleware/websocket/client.rb

```diff
@@ -1,10 +1,13 @@
 module Websocket
   class Client
-    attr_reader :connection_client, :client_id
+    attr_reader :connection_client, :client_attributes
+    attr_accessor :position
 
     def initialize(connection_client: client, client_id: client_id)
       @connection_client = connection_client
       @client_id = client_id
+      @position = '0%'
```

Comment:
I'd rather we used an integer for this. We'll end up in a lot of trouble should we ever need to rely up this value for anything other than display purposes.

Reply:
@jgayfer good call, fixed.